src/frontends/tests/onnx/onnx_import.in.cpp

```diff
@@ -5179,3 +5179,32 @@ NGRAPH_TEST(${BACKEND_NAME}, onnx_grid_sample) {
 
     test_case.run();
 }
+
+NGRAPH_TEST(${BACKEND_NAME}, onnx_model_layer_norm) {
+    const auto function = onnx_import::import_onnx_model(file_util::path_join(SERIALIZED_ZOO, "onnx/layer_norm.onnx"));
+
+    auto test_case = test::TestCase(function, s_device);
+    std::vector<float> a = {
+        31.,  245.,   47.,  239.,
+        -106.,  167.,   33.,  157.,
+        59., -193., -103., -246.,
+    };
+
+    std::vector<float> b = {
+        22.538681,  -13.113842, -144.41461 ,   69.15499,
+        14.064551,  -19.023893, -107.303635,   62.184105,
+        72.52179 , -125.468506,  -83.254326,  -51.877796,
+    };
+
+    std::vector<float> bias = {43., -83., -92.,  12.,};
+
+    std::vector<float> scale = {19., 68., 57., 59.,};
+
+    test_case.add_input<float>(Shape{3, 4}, a);
+    test_case.add_input<float>(scale);
+    test_case.add_input<float>(bias);
+    test_case.add_expected_output<float>(Shape{3, 4}, b);
+
+    test_case.run_with_tolerance_as_fp(1e-5);
+}
+
```

Comment:
Please add more tests, in particular:
- with an axis different than the default
- perhaps a different value of `epsilon`
The examples in the operator's spec give some ideas of what could be tested for this op.